### DISCUSSION TITLE
se puede usar en programas de windows fuera del navegador?

### DISCUSSION THREAD

**Discussion by juanx01x** [ruffle-rs/ruffle#3347] [General]
descargué la versión de escritorio porque quiero usar un simulador de ensamblaje de PC pero al momento de querer instalar Ruffle me pide que lo guarde en una carpeta con una extencion un poco rara. no se si alguien pueda mandarme a un tutorial o que tengo que hacer

**Comment by ousia** [comment]
@juanx01x,

¿de dónde descargas _Ruffle_ para _Windows_?

En https://github.com/ruffle-rs/ruffle/releases descarga el archivo que termina en `windows.zip`.

Cuando lo he usado (habitualmente uso _Linux_), contiene una versión ejecutable para _Windows_, que no tiene nada que instalar.

En el directorio donde lo descomprimas, ahí lo tendrás. De ahí lo podrás mover donde quieras.

La extensión es `.exe`. De hecho, no deberías poder verla.

Perdóname si me equivoco, pero quizá no estés descargando la versión de _Windows_.

Espero que te sirva. Si no fuera así, dime qué falla.

Por cierto, el icono de confusión es porque no entiendo qué es lo que te puede estar fallando.
